mmhuman3d/data/data_converters/eft.py

```diff
@@ -15,7 +15,7 @@
 @DATA_CONVERTERS.register_module()
 class EftConverter(BaseModeConverter):
     """Eft dataset `Exemplar Fine-Tuning for 3D Human Pose Fitting Towards In-
-    the-Wild 3D Human Pose Estimation' 3DV'2021 More details can be found in
+    the-Wild 3D Human Pose Estimation' 3DV'2020 More details can be found in
```

Comment:
It seems they publied on 3DV'2021 https://github.com/facebookresearch/eft

Reply:
changed in de676df